src/error.rs

```diff
@@ -1833,32 +1872,33 @@ impl IntoDiagnostics<FileId> for TypecheckError {
                     // information, so we just ignore it.
                     TypecheckError::TypeMismatch(_, _, _) => (),
                     err => {
-                        diags.extend(err.into_diagnostics(files, stdlib_ids).into_iter()
-                            .map(|mut diag| {
-                                diag.message = format!("While matching function types: {}", diag.message);
+                        diags.extend(err.into_diagnostics(files, stdlib_ids).into_iter().map(
+                            |mut diag| {
+                                diag.message =
+                                    format!("While matching function types: {}", diag.message);
```

Comment:
I'm not sure how exactly this message is formatted, but maybe the "While" should be lowercase here as well, lik on line 1826.